dtfs-central-api/src/v1/routes/utilisation-reports-routes.js

```diff
@@ -1063,9 +1063,9 @@ utilisationReportsRouter
  * @openapi
  * /utilisation-reports/:reportId/fee-records/:feeRecordId/correction-request-review/:userId:
  *   get:
- *     summary: Get correction request review information to check before sending
+ *     summary: Get correction request review information to check before sending or error key if already submitted
  *     tags: [Utilisation Report]
- *     description: Get correction request review information to check before sending
+ *     description: Get correction request review information to check before sending or error key if already submitted
```

Comment:
Can't comment on the line itself but we need to update the response schema on lines 1093-1095

Reply:
removed and added comment in correct place